deployment/deployutils/wizardInputs.cpp

```diff
@@ -86,13 +86,26 @@ void CWizardInputs::setEnvironment()
   if(m_pXml->hasProp("@ipList"))
     formIPList(m_pXml->queryProp("@ipList"), m_ipaddress);
 
+  if(m_pXml->hasProp("@roxieNodes"))
+    m_roxieNodes = atoi(m_pXml->queryProp("@roxieNodes"));
+
+  if(m_pXml->hasProp("@thorNodes"))
+    m_thorNodes = atoi(m_pXml->queryProp("@thorNodes"));
+
+  m_espNodes = 1;
+  if(m_pXml->hasProp("@espNodes"))
+    m_espNodes = atoi(m_pXml->queryProp("@espNodes"));
+
   if(m_pXml->hasProp("@supportNodes"))
   {
     m_supportNodes = atoi(m_pXml->queryProp("@supportNodes"));
 
     if (m_supportNodes)
     {
-      if (m_ipaddress.length() > 0 && m_ipaddress.length() > m_supportNodes)
+      if (m_ipaddress.length() > 0 &&
+         ((m_ipaddress.length() > m_supportNodes) ||
+         ((m_ipaddress.length() == m_supportNodes) &&
+          (m_roxieNodes == 0) && (m_thorNodes == 0) && (m_espNodes == 0))))
```

Comment:
This is where I'd worry about m_roxieNodes and m_thorNodes not being initialized.